libs/coin-modules/coin-solana/src/synchronization.ts

```diff
@@ -394,6 +394,14 @@ export const getAccountShapeWithAPI = async (
     }
     return nfts;
   }, [] as ProtoNFT[]);
+  let cleanOperations: Operation[];
+  if (!coinConfig.getCoinConfig().ShowNFTs) {
+    cleanOperations = mainAccTotalOperations.filter(
```

Comment:
Should be done there : https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/DataModel.ts#L40